benchmarks/test_distances.py

```diff
@@ -46,7 +46,7 @@
 def Neff_from_N_ur_N_ncdm(N_ur, N_ncdm):
     """Calculate N_eff from the number of relativistic
     and massive neutrinos."""
-    Neff = N_ur + N_ncdm * ccl.physical_constants.TNCDM**4 / (4./11.)**(4./3.)
+    Neff = N_ur + N_ncdm * ccl.physical_constants.T_ncdm**4 / (4./11.)**(4./3.)
```

Comment:
since this is gonna go away in v3, change so we use the v3 way

Reply:
oops I missed that, thanks

Reply:
I need your input here. As you might have seen, there are several places where we need the value of `T_ncdm` where `cosmo` is not passed. We could either copy+paste the same value everywhere, which will make it hard to maintain, or we can have it stored somewhere, in a module, and access it from there.

I chose to leave it in `physical_constants` as that was its original place. Maybe a more fitting location would be in `core.py`, i.e. name a variable `T_ncdm = ...` and when we need it, do `from pyccl.core import T_ncdm`. However, I think this option would be a little non-intuitive.

I propose we leave it in the constants, or create a separate class in `parameters.py` which holds things like that, and would be accessed via `ccl.special_params.T_ncdm`.